packages/MachineLearning/tests/Unit/AiRuntimeContractsTest.php

```
@@ -0,0 +1,326 @@
+
+        $definitionA = new AiCapabilityDefinition(
+            featureKey: 'procurement_ai_quote_summary',
+            capabilityGroup: AiCapabilityGroup::DOCUMENT_INTELLIGENCE,
+            requiresAi: true,
+            hasManualFallback: true,
+            fallbackUiMode: AiFallbackUiMode::SHOW_MANUAL_CONTINUITY_BANNER,
+            degradationMessageKey: 'ai.procurement.quote_summary.degraded',
+            operatorCritical: true,
+        );
+
+        $endpoint = new AiEndpointHealthSnapshot(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$endpoint'. (undefined)

(UnusedLocalVariable)